about.html

```diff
@@ -0,0 +1,168 @@
+            <h3>Kadoma Community Development</h3>
+            <div class="underline">
+                <hr>
+            </div>
+            <p>
+                Kadoma Community Development aims to share benefits to the community in which the people live in,
+                We welcome any suggestions whether positive or negative
+            </p>
+            <div class="logo-design">
+                <img class="header-img" src="img/work.png" alt="logo">
+                <p>
+                    Kadoma <br> Community Development
+                </p>
+            </div>
+        </section>
+        <section class="past-summit d-flex flex-column justify-content-center align-items-center mb-5">
+            <h3 class="mt-5">See Past Projects</h3>
+            <div class="underline">
+                <hr>
+            </div>
+            <p>Take a look at the past successful projects</p>
+            <div class="row d-flex justify-content-center">
+                <div class="col-md-5 justify-content-center align-items-center d-flex mb-3 ward-34 flex-column mx-1 col-10 ">
+                    <h2>2019</h2>
+                    <p class="text-center">Ward 13 Projects</p>
+                </div>
+                <div class="col-md-5 justify-content-center align-items-center d-flex flex-column mb-3 ward-40 col-10 mx-1">
+                    <h2>2017</h2>
+                    <p class="text-center">Ward 34 projects</p>
+                </div>
+            </div>
+        </section>
+    </div>
+    <footer>
+        <div class="container-fluid footer-bg partner">
+            <section class="container d-flex flex-column justify-content-center align-items-center">
+                <div class="partner-header d-flex flex-column justify-content-center align-items-center">
+                    <h3 class="partner-title">Partner</h3>
+                    <div class="underline"><hr></div>
+                </div>
+                <ul class="d-flex list-unstyled flex-wrap justify-content-around col-12">
+                    <li class="partnerimj"><img src="img/JetBlueLogo.png" alt="partner1"></li>
+                    <li class="partnerimj"><img src="img/logitech_logo.png" alt="partner2"></li>
+                    <li class="partnerimj"><img src="img/P_G_logo.png" alt="partner3"></li>
+                    <li class="partnerimj"><img src="img/ypf_logo.png" alt="partner4"></li>
+                    <li class="partnerimj"><img src="img/P_G_logo.png" alt=""></li>
+                </ul>
+            </section>
+        </div>
+        <section class="footer">
+            <div class="logitep">
+                <div class="footlogo">
+                    <img class="header-img" src="img/work.png" alt="logo">
+                     <h5>Kadoma Community</h5>
+                </div>
+                <p class="footerp">2021 Illuminations. some right reserved </p>
+            </div>
+        </section>
+    </footer>
```

Comment:
- [ ] On desktop view, the footer has a dark background whereas on mobile view it has a white background. Please kindly fix that.
- [ ] Please add the footer to the homepage on mobile view. I know that the original design is not present but you have to add it because all the elements of the page should be present on desktop view as well as on mobile view

Reply:
oooh how did i forget that unfortunately i dont have any reviews left.

Reply:
can you please just put it in the optional section as i dont have any other review requests left. i will be sure to add that

Reply:
my bad i have to restart the module then